odoo/addons/base/models/ir_rule.py

```diff
@@ -230,11 +203,9 @@ def _make_access_error(self, operation, records):
         resolution_info = _("Contact your administrator to request access if necessary.")
 
         if not self.env.user.has_group('base.group_no_one') or not self.env.user.has_group('base.group_user'):
-            msg = """{operation_error}
+            msg = f"""{operation_error}
 
-{resolution_info}""".format(
-                operation_error=operation_error,
-                resolution_info=resolution_info)
+{resolution_info}"""
```

Comment:
Could be even simpler
```python
msg = f"{operation_error}\n\n{resolution_info}"
```

Reply:
Yeah why not ? Should I change the other one ?
```python
f"""{operation_error}

{failing_records}
{failing_user}

{failing_rules}

{resolution_info}"""
```
into `f"{operation_error}\n\n{failing_records}\n{failing_user}\n\n{failing_rules}\n\n{resolution_info}"`

Reply:
The first one is simple enough to remain readable when on a single line, but I am not sure about that one...